src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt

```diff
@@ -71,17 +71,27 @@ def CreatePythonPathEntries(python_imports, module_space):
 
 # Find the runfiles tree
 def FindModuleSpace():
-  stub_filename = os.path.abspath(sys.argv[0])
-  module_space = stub_filename + ('.exe' if IsWindows() else '') + '.runfiles'
-  if os.path.isdir(module_space):
-    return module_space
-
-  runfiles_pattern = "(.*\.runfiles)/.*"
-  if IsWindows():
-    runfiles_pattern = "(.*\.runfiles)\\.*"
-  matchobj = re.match(runfiles_pattern, os.path.abspath(sys.argv[0]))
-  if matchobj:
-    return matchobj.group(1)
+  stub_filename = sys.argv[0]
+  if not os.path.isabs(stub_filename):
+    stub_filename = os.path.join(os.getcwd(), stub_filename)
+
+  while True:
+    module_space = stub_filename + ('.exe' if IsWindows() else '') + '.runfiles'
+    if os.path.isdir(module_space):
+      return module_space
+
+    runfiles_pattern = '(.*\.runfiles)' + ('\\' if IsWindows() else '') + '.*'
```

Comment:
Yes, '/' was missing from the else branch.
Using os.path.sep does not work because '\' needs to be escaped.  In fact, it appears that the
regular expression for Windows was incorrect before.  We need to use '\\\\' in the string literal
so that the regexp matcher sees '\\'.

Reply:
My comment was mangled.  Let's try again:

Yes, '/' was missing from the else branch.
Using os.path.sep does not work because '\\' needs to be escaped. In fact, it appears that the
regular expression for Windows was incorrect before. We need to use '\\\\\\\\' in the string literal
so that the regexp matcher sees '\\\\'.